EIPS/eip-3041.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 3041
+title: Add basefee in eth_getBlockByHash response
+author: Abdelhamid Bakhta (@abdelhamidbakhta)
+discussions-to: https://ethereum-magicians.org/t/eip-3041-add-basefee-in-eth-getblockbyhash-response/4825
+status: Draft
+type: Standards Track
+category : Interface
+created: 2020-10-13
+requires: 1559
+---
+
+## Simple Summary
+Add basefee field in `eth_getBlockByHash` RPC endpoint response.
+
+## Abstract
+[EIP-1559](/EIPS/eip-1559) introduces a base fee per gas in protocol.
+This value is maintained under consensus as a new field in the block header structure.
+This change should be reflected in RPC methods that return block header information.
```

Comment:
```suggestion
Adds `baseFee` property to the `eth_getBlockByHash` JSON-RPC request `result` object.  This property will contain the value of the base fee for any block after the EIP-1559 fork.
```
Abstract should just be a human-readable version of the specification that is particularly terse.  It won't be as detailed as the specification section, and it likely won't mention edge cases or discuss minor details about the specification.  It should target a human reader trying to get a high level understanding of the specification, while the specification section is targeting implementers.